timeseries/tests/test_SimulatedTimeSeries.py

```diff
@@ -22,16 +23,39 @@ def test_produce():
         ts = SimulatedTimeSeries(data)
         result = list(ts.produce(chunk_size))
         expect_result = []
-        chunk = []
-        for time, value in enumerate(make_simple_data(data_size)):
-            if len(chunk) >= chunk_size:
-                expect_result.append(chunk)
-                chunk = []
-            chunk.append((time, value))
-        if chunk:
-            expect_result.append(chunk)
-        assert result == expect_result
+        for time, value in enumerate(make_simple_data(data_size)):           
+            if len(expect_result) >= chunk_size:
+                assert result == expect_result
+                result = list(ts.produce(chunk_size))
+                expect_result = []
+            expect_result.append((time, value))
 
+def test_mean():
+    finite_data = make_simple_data(1000)
+    ts = SimulatedTimeSeries(finite_data)
+    assert ts.mean() == 500
+
+    infinite_data = make_simple_data(3000)
+    ts = SimulatedTimeSeries(infinite_data)
+    with raises(ValueError):
+        ts.mean()
+
+def test_std():
+    finite_data = make_simple_data(500)
+    ts = SimulatedTimeSeries(finite_data)
+    finite_data_copy = make_simple_data(500)
+    data_list = []
+    for time, value in enumerate(finite_data_copy):
+        data_list.append(value) 
+    expect_result = np.std(data_list)
+    assert round(ts.std()) == round(expect_result.round())
```

Comment:
Use threshold expression.
In fact, round(3.55555)=4, comparison under this level is too imprecise